src/sql/connection/connection.py

```diff
@@ -208,7 +208,7 @@ def set(
             attributes. As of 0.9.0, only the autocommit option is needed.
         """
         connect_args = connect_args or {}
-        SWITCH_PREFIX = "Switching to connection"
+        SWITCH_PREFIX = "Switching to Connection"
```

Comment:
connection shouldn't be capitalized

Reply:
[changed](https://github.com/ploomber/jupysql/pull/796/files#diff-62e3d6289e3242b271e6df3a7f201ab003f8ce73a5daf1a824cc2b4d892513d1R230-R242)